index.js

```diff
@@ -294,10 +324,22 @@ module.exports = function (schema, options) {
             doc.deletedId = params.deletedId 
         }
 
-        return updateDocumentsByQuery(this, conditions, doc, callback);
+        const options = {};
+        if (params.session) {
+            options.session = params.session;
+        }
+
+        return updateDocumentsByQuery(this, conditions, doc, options, callback);
```

Comment:
🤏 You should be able to have options set to undefined and still have this work with the code at the top
```suggestion
        var options;
        if (params.session) {
            options = { session: params.session };
        }

        return updateDocumentsByQuery(this, conditions, doc, options, callback);
```